<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Maxper</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>dpsf</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>halfbox</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxPLIM"/>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxDPSF"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>max size</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>saddle</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>plim</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Maximum Peaks</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>dlim</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxDLIM"/>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxSaddle"/>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinBoxMaxper"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Sigma</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="horizontalSliderMaxPeaks">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="spinBoxHalfBox"/>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="spinBoxMaxSize"/>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxSigma"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonAccept">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Accept Selection</string>
        </property>
        <property name="icon">
         <iconset resource="../VSOP.qrc">
          <normaloff>:/images/blueberry/check_2.png</normaloff>:/images/blueberry/check_2.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonExtract">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Extract Sources</string>
        </property>
        <property name="icon">
         <iconset resource="../VSOP.qrc">
          <normaloff>:/images/blueberry/bonus.png</normaloff>:/images/blueberry/bonus.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="../VSOP.qrc">
          <normaloff>:/images/blueberry/close_delete_2.png</normaloff>:/images/blueberry/close_delete_2.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../VSOP.qrc"/>
 </resources>
 <connections/>
</ui>
